service/src/test/java/org/kiwiproject/champagne/resource/HostConfigurationResourceTest.java

```diff
@@ -155,14 +160,135 @@ void shouldNotAuditDeleteIfDoesNotChangeAnything() {
         }
     }
 
-    private void verifyAuditRecorded(long id, Action action) {
+    private void verifyAuditRecorded(long id, Class<?> recordClass, Action action) {
```

Comment:
Suggest renaming to `auditRecordClass` to avoid confusion; I initially thought it was talking about the `Class` of a Java `record`, then quickly realize it was related to audit records. But might as well make it unambiguous.